### PR TITLE
autoshotgun changes

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
Buffs the autoshotguns flechette scatter to make it more useful at mid range, and gives it accuracy bonus so it can be used attachless more effectively. Slugs slow a bit more. It has 20 force now because it has an integrated stock.

## Why It's Good For The Game
Autoshotgun is really bad right now. Like unuseably bad. This tries to make it more in line with other pfc guns.
Buckshot is in a bad place. This is a test to make it maybe slightly better.

## Changelog
:cl:
balance: TX-15 has more natural accuracy,flech has less spread does less damage,slugs slow more and deal 10 more damage. Has 20 force on normal melee hits. Firerate increased by 100ms.
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -4. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

File: code/modules/projectiles/guns/rifles.dm [hunk 1]
```diff
@@ -603,6 +603,7 @@
 	dry_fire_sound = 'sound/weapons/guns/fire/shotgun_empty.ogg'
 	caliber = "16 gauge" //codex
 	max_shells = 12 //codex
+	force = 20.0
```
Comment (Rohesie) [inline code/modules/projectiles/guns/rifles.dm:606]:
```suggestion
	force = 20
```
Don't add decimal zeros.

**Comment by Rohesie** [comment]
People have been complaining that it with flech is a bit too powerful.

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-8). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**